src/index.js

```diff
@@ -23,6 +23,9 @@ class OptimizeCssAssetsWebpackPlugin extends LastCallWebpackPlugin {
     this.options.cssProcessorOptions = !options || options.cssProcessorOptions === undefined ?
```

Comment:
Is this still being used?

Reply:
yes， { from, to }


> As a convenience, plugins also expose a process method so that you can use them as standalone tools.
```
cleaner.process(css, processOpts, pluginOpts)
// This is equivalent to:
postcss([ cleaner(pluginOpts) ]).process(css, processOpts)